### PR TITLE
jst token

### PR DESCRIPTION
**Pull request by git-harikrishna**
login using jwt token for accessing the options

### CODE REVIEW

File: controller/userController.js [hunk 1]
```diff
@@ -79,7 +82,7 @@ exports.updateUser = async (req, res, next) => {
 
     console.log(user.name);
     console.log(user);
-
+    if (!(req.user.id == user.id)) return res.json({ msg: "Unauthorized" });
     const updatedUser = await user.save();
 
     res.json(updatedUser);
```
Comment (sreelakshmi-thudupattil) [inline controller/userController.js:88]: While returning a response, include the appropriate status code. Please carry out this action for each response.

File: controller/userController.js [hunk 2]
```diff
@@ -106,3 +109,16 @@ exports.deleteUser = async (req, res, next) => {
     res.send("Error: " + e);
   }
 };
+
```
Comment (sreelakshmi-thudupattil) [inline controller/userController.js:112]: Remove unnecessary code and console.logs

File: controller/authenticationController.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+const jwt = require("jsonwebtoken");
+
+exports.login = async (req, res, next) => {
+  try {
+    console.log("login called");
+    const name = req.body.name;
+    const id = req.body.id;
```
Comment (sreelakshmi-thudupattil) [inline controller/authenticationController.js:7]: please remove id; name and password should be used for login.

File: controller/userController.js [hunk 4]
```diff
@@ -27,8 +27,7 @@ exports.addUser = async (req, res, next) => {
 };
 
 exports.getUser = async (req, res, next) => {
```
Comment (sreelakshmi-thudupattil) [inline controller/userController.js:29]: While hitting this API , am getting multiple user details, Please change this implementation as we discussed over call

File: .env [hunk 5]
```diff
@@ -1,4 +1,9 @@
 # API
 password=PuJznXDXengqqHNM
+dbUrl = "mongodb+srv://harikrish24work:summa123@cluster0.bpj1clt.mongodb.net/test?retryWrites=true&w=majority"
 # Database
 # DATABASE_NAME=myDatabaseName
+
+ACCESS_TOKEN_SECRET = af30b417efe5ea4c9b08efd075ef51796fe79696fdec8875d276663175ea73e58fab8aff093b504df77bbeb8810737e4c3c5e4a70932030daf8d852aba97af96
```
Comment (sreelakshmi-thudupattil) [inline .env:7]: Please  dont store access token/refresh token in env file

File: controller/userController.js [hunk 6]
```diff
@@ -61,8 +60,12 @@ exports.getUserById = async (req, res, next) => {
 exports.updateUser = async (req, res, next) => {
   console.log("updateUser called");
```
Comment (sreelakshmi-thudupattil) [inline controller/userController.js:61]: use access token to identify the user and update details

File: .env (hunk 5 above)
Comment (git-harikrishna) [reply .env:7]: sree those are token secrets which are used to generate access token and the access tokens are not stored anywhere so should i remove it ?

**Pull request closed without merging**